main.py

```diff
@@ -21,12 +27,13 @@ class EmailScheduleResponse(BaseModel):
 
 
 @app.get("/")
-def read_root():
-    return {"Hello": "World"}
+def read_root(request: Request):
+    return templates.TemplateResponse("email_schedule_form.html", {"request": request})
 
 
 @app.post("/email-schedules", response_model=EmailScheduleResponse)
-def create_email_schedule(email_schedule_request: EmailScheduleRequest):
+def create_email_schedule(request: Request, email_schedule_request: EmailScheduleRequest):
```

Comment:
Why do we need `request: Request` here?

Reply:
The `request: Request` parameter is not necessary in the `create_email_schedule` function. It seems to be a mistake. I will remove it.

Reply:
The `request: Request` parameter is not necessary in the `create_email_schedule` function. It seems to be a mistake. I will remove it. Thank you for pointing it out.